units/generate_tasks.go

```diff
@@ -129,6 +129,13 @@ func (j *generateTasksJob) generate(ctx context.Context, t *task.Task) error {
 	})
 	start = time.Now()
 
+	if t.Project == "mms" {
+		grip.Info(message.Fields{
+			"ticket":    "EVG-12967",
+			"generator": t.Id,
+			"projects":  projects,
```

Comment:
This has the potential to be pretty big, doesn't it? And this would run for any generated task on cloud which seems like a lot. Is there a way to tighten this up?

Reply:
It is, but the reason I'm logging the entire thing is because I'm planning on using most of it. Since we know that this will just get called once per generator, I think splunk can handle it